userspace/libsinsp/parsers.cpp

```diff
@@ -1349,13 +1349,6 @@ void sinsp_parser::parse_clone_exit_caller(sinsp_evt *evt, int64_t child_tid) {
 		return;
 	}
 
-	/* Refresh user / loginuser / group */
-	if(new_child->m_container_id.empty() == false) {
```

Comment:
No need to refresh user/group anymore when container id changes.
User group manager does now attach the `subscribe_on_new_container` listener on the container manager to cache all new containers users.